lib/ads/ad_manager.js

```diff
@@ -902,6 +902,13 @@ shaka.ads.AdManager.AD_METADATA = 'ad-metadata';
  */
 shaka.ads.AdManager.AD_RECOVERABLE_ERROR = 'ad-recoverable-error';
 
+/**
+ * The event name for when the ad manager dispatch errors.
+ *
+ * @const {string}
+ * @export
+ */
+shaka.ads.AdManager.AD_ERROR = 'ad-error';
```

Comment:
You should also write an `@event` JSDoc entry at the top of the file. That's what is used to generate the docs for the event.

Reply:
Done!